pkg/operator/controllers/autosizednodes/autosizednodes_controller.go

```diff
@@ -32,6 +32,10 @@ type Reconciler struct {
 const (
 	ControllerName = "AutoSizedNodes"
 	configName     = "dynamic-node"
+
+	// machineconfiguration annotation added on the kubelet CRD - dynamic-node
+	mcAnnotationName  = "machineconfiguration.openshift.io/mc-name-suffix"
+	mcAnnotationValue = ""
```

Comment:
I would comment why we use `""` here just so we remember in a while, it's unintuitive.